code/modules/atmospherics/machinery/components/unary_devices/vent_pump.dm

```diff
@@ -28,23 +28,59 @@
 	// ATMOS_INTERNAL_BOUND: Do not pass internal_pressure_bound
 	// NO_BOUND: Do not pass either
 
+	///id of air sensor its connected to
+	var/chamber_id
+
 /obj/machinery/atmospherics/components/unary/vent_pump/New()
 	if(!id_tag)
 		id_tag = SSnetworks.assign_random_name()
+		var/static/list/multitool_tips = list(
```

Comment:
Nit: Should be named something more generic like `tool_screentips` in case we change this out from just multitools

Reply:
Got it